Detectors/TPC/reconstruction/include/TPCReconstruction/ClusterHardware.h

```diff
@@ -0,0 +1,46 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file ClusterHardware.h
+/// \brief Class of a TPC cluster as produced by the hardware cluster finder (needs a postprocessing step to convert into ClusterNative
+/// \author David Rohr
+#ifndef ALICEO2_TPC_CLUSTERHARDWARE_H
+#define ALICEO2_TPC_CLUSTERHARDWARE_H
+
+namespace o2 { namespace TPC{
+
+struct ClusterHardware { //Temporary draft of hardware clusters. The ...Pre members are yet to be defined, and will most likely either be floats or fixed point integers.
+  float mPadPre;                //Quantity needed to compute the pad
+  float mTimePre;               //Quantity needed to compute the time
+  float mSigmaPad2Pre;          //Quantity needed to compute the sigma^2 of the pad
+  float mSigmaTime2Pre;         //Quantity needed to compute the sigma^2 of the time
+  unsigned short mQMax;         //QMax of the cluster
+  unsigned short mQTot;         //Total charge of the cluster
+  unsigned char mRow;           //Row of the cluster (local, needs to add PadRegionInfo::getGlobalRowOffset
+  unsigned char mFlags;         //Flags of the cluster
```

Comment:
I think you have to use `//<` to let doxygen know the comment is for what is written in front. Same below for all new code